src/workerd/io/worker-fs.h

```diff
@@ -694,8 +694,17 @@ class TmpDirStoreScope final {
     return dir.addRef();
   }
 
+  kj::PathPtr getCwd() const {
+    return kj::PathPtr(cwd);
+  }
+
+  void setCwd(kj::Path newCwd) const {
```

Comment:
Why do we have const in here?

```suggestion
  void setCwd(kj::Path newCwd) {
```

Reply:
Went ahead and removed the const usage.